src/org/spdx/rdfparser/license/SimpleLicensingInfo.java

```diff
@@ -305,6 +332,12 @@ public void populateModel() throws InvalidSPDXAnalysisException {
 		} else {
 			setPropertyValue(SpdxRdfConstants.RDFS_NAMESPACE, SpdxRdfConstants.RDFS_PROP_SEE_ALSO, seeAlso);
 		}
+		// SeeAlsoDetails
+		if (seeAlsoDetails == null) {
+			removePropertyValue(SpdxRdfConstants.RDFS_NAMESPACE, SpdxRdfConstants.RDFS_PROP_SEE_ALSO_DETAILS);
```

Comment:
Similar comment on which constant to use

Reply:
I have corrected this